New UI to change password

rodrigosouzalopes94 opened the pull request:
## Issue Fix
Fixes #{Issue Number}
Jira Task: [MW145]https://mifosforge.jira.com/browse/MW-148



## Screenshots
| New Screen | Old Screen |
|----------------------------------------------------------------------------------------|-------------------------------------------|
|  ![print_MW145](https://github.com/user-attachments/assets/f46f545d-e7a2-475f-8c26-66fd207f3ef1)  |  ![old_screen_password](https://github.com/user-attachments/assets/198bef21-aebe-485c-9be8-bda7b1212330) | 


## Description
A new change password screen

##
<!--Please make sure these boxes are checked before submitting your pull request - thanks!-->

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

rodrigosouzalopes94 committed:
New UI to change password

niyajali commented on core/designsystem/src/main/kotlin/org/mifospay/core/designsystem/component/TextField.kt:
```diff
@@ -111,12 +111,18 @@ fun MfPasswordTextField(
-        label = { Text(label) },
+        label = { Text(
+            label,
+            color = NewUi.primaryColor,
```
Can you format these lines of code.

rodrigosouzalopes94 replied:
I already change

niyajali commented on core/designsystem/src/main/kotlin/org/mifospay/core/designsystem/component/TextField.kt:
```diff
@@ -111,12 +111,18 @@ fun MfPasswordTextField(
     errorMessage: String? = null,
+
 ) {
+
```
Can you remove blank lines

rodrigosouzalopes94 replied:
Done

niyajali commented on core/designsystem/src/main/kotlin/org/mifospay/core/designsystem/component/TextField.kt:
```diff
@@ -132,9 +138,16 @@ fun MfPasswordTextField(
+            focusedBorderColor = Color.Transparent, // Cor da borda quando focado
+            unfocusedBorderColor = Color.Transparent, // Cor da borda quando não focado
+            errorBorderColor = Color.Transparent, // Cor da borda quando em erro
+            disabledBorderColor = Color.Transparent, // Cor da borda quando desabilitado
```
And remove these comments too

rodrigosouzalopes94 replied:
Done

niyajali commented on core/designsystem/src/main/kotlin/org/mifospay/core/designsystem/component/TextField.kt:
```diff
@@ -301,8 +314,12 @@ fun MfOutlinedTextFieldPreview() {
         Box(
+
             modifier = Modifier.background(color = Color.White),
+
```
Remove these extra blank lines..

rodrigosouzalopes94 replied:
Done

niyajali commented on feature/editpassword/src/main/kotlin/org/mifospay/feature/editpassword/EditPasswordScreen.kt:
```diff
@@ -130,9 +138,10 @@ private fun EditPasswordScreen(
-                )
+                        .fillMaxWidth(0.9f)
+                        .padding(vertical = 8.dp),
+
```
Remove the blank lines

rodrigosouzalopes94 replied:
Done

niyajali commented on feature/editpassword/src/main/kotlin/org/mifospay/feature/editpassword/EditPasswordScreen.kt:
```diff
@@ -175,29 +185,47 @@ private fun EditPasswordScreen(
                     )
+                    Spacer(modifier = Modifier.height(8.dp))
+
+
```
Remove here too

niyajali commented:
Run `ci-prepush.sh` bash script before commiting or pushing your changes it will format and check for any errors

niyajali commented:
Can you upload both Old UI and Updated UI Image

niyajali commented:
Commit your changes, i didn't see any

rodrigosouzalopes94 committed:
Fixed some blank spaces and clean comments

Some fixed blank spaces and remove comments

Delete git file

Fixed the password screen

niyajali commented on git:
Remove this file

rodrigosouzalopes94 replied:
Done

niyajali commented:
@rodrigosouzalopes94 fix this build error

> Task :core:designsystem:compileDemoDebugKotlin FAILED
e: file:///home/runner/work/mobile-wallet/mobile-wallet/core/designsystem/src/main/kotlin/org/mifospay/core/designsystem/component/TextField.kt:119:25 Unresolved reference 'NewUi'.

itsPronay reviewed (changes requested):
Hello @rodrigosouzalopes94 , thanks for the PR but You modified the old `OutlinedTextField` and set colors to `color.Transparent`. We won't be using that anymore. 

Take reference from finance/payment/editProfile screen. also after you are done please attach the figma UI as well

rodrigosouzalopes94 commented:
> Hello @rodrigosouzalopes94 , thanks for the PR but You modified the old `OutlinedTextField` and set colors to `color.Transparent`. We won't be using that anymore.
> 
> Take reference from finance/payment/editProfile screen. also after you are done please attach the figma UI as well

Can you help to make the clean code about these styles?

itsPronay commented:
> Can you help to make the clean code about these styles?

it's not about styles, You need to modify `MifosTextField` that we already have in `textfield.kt` file.
 If you still have any doubts feel free to let me know on Slack.

rodrigosouzalopes94 commented:
Done.

itsPronay commented:
> Done.

1. please update the new UI in your commit message. also, add Figma UI as well
2. fix conflicts
3. Please squash your commits as well (squash before pull the remote changes) 
![image](https://github.com/user-attachments/assets/11daabb9-a89b-4961-b603-08e870581f13)

rodrigosouzalopes94 committed:
Stashing my commits

rodrigosouzalopes94 commented:
Figma UI
![image](https://github.com/user-attachments/assets/aaeef135-0b60-46e9-99b0-a1c40b9e1d5a)
New Screen
![newUI](https://github.com/user-attachments/assets/db1680a8-b99f-4b91-8648-59be47f09d15)

rodrigosouzalopes94 reviewed (commented):
Fixed the conflits

niyajali commented:
@rodrigosouzalopes94 As I can see, the trailing icon color doesn't match the color as per Figma UI

rodrigosouzalopes94 commented:
I gonna fixed.

niyajali commented:
@rodrigosouzalopes94 take a look into this pr #1787 , might helps

therajanmaurya commented:
@niyajali can you help @rodrigosouzalopes94 so we can merge this PR

niyajali commented:
@rodrigosouzalopes94 Here is the pr #1787 of this module

therajanmaurya commented:
Already Implemented
